tools/serverpod_cli/lib/src/util/string_validators.dart

```diff
@@ -16,4 +16,7 @@ class StringValidators {
 
   static bool isValidTableIndexName(String name) =>
       _mixedSnakeCaseTester.hasMatch(name);
+
+  static bool isValidProjectName(String name) =>
+      RegExp(r'^[a-z]+(?:[0-9a-z]+|_[0-9a-z]+)*$').hasMatch(name);
```

Comment:
Please add a description on the valid format of the regex, could you also add some unit tests to validate these rules.

Reply:
BTW can I get the unit test folder path.

Reply:
There is a `test` folder in the serverpod_cli project!